### PR TITLE
[ios] 12791 added initial test to public {WIP}

### PR DESCRIPTION
**Pull request by adam-govan**
{Work in Progress}

currently has the following testing:
- SwrveTrigger
- SwrveTriggerCondition

### CODE REVIEW

File: SwrveSDKTest/Podfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+target 'SwrveSDKTest' do
+  pod 'SwrveSDKCommon', :path => '../'
+  pod 'SwrveConversationSDK', :path => '../'
+  pod 'SwrveSDK', :path => '../'
+  #pod 'SwrveSDK', '~> 4.5.0'
```
Comment (Sergio-Mira) [inline SwrveSDKTest/Podfile:5]: I don't think the tests will ever point to the live cocoapods

Comment (adam-govan) [inline SwrveSDKTest/Podfile:5]: This would be handy to test on release. We can run all the tests against the latest version.

**Comment by Sergio-Mira** [comment]
👍

File: SwrveSDKTest/Podfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+end
+
+target 'SwrveSDKTestTests' do
```
Comment (dominicmarmionswrve) [inline SwrveSDKTest/Podfile:8]: SwrveSDKTestTests?

**Pull request closed without merging**